test/core/libp2p/peer/address_repository/inmem_address_repository_test.cpp

```diff
@@ -49,6 +49,11 @@ struct InmemAddressRepository_Test : public ::testing::Test {
       Multiaddress::create("/ip4/127.0.0.1/tcp/8082").value();
   const Multiaddress ma4 =
       Multiaddress::create("/ip4/127.0.0.1/tcp/8083").value();
+
+  template <typename... T>
+  std::vector<T...> vec(T &&... arg) {
```

Comment:
don't see where it is needed

Reply:
It is just a helper, so I don't need to write long vector inialization